applications/chat_with_rag/blocks_llm_chat_with_rag.py

```
@@ -11,7 +11,7 @@
 
 from demos.tool_calling.tool_descriptors import tools_rag_descriptor
 # noinspection PyUnresolvedReferences
-from fn_rag import lookup_in_documentation
+from fn_rag import lookup_in_documentation, list_documents
```

Ruff (0.8.2)

`fn_rag.lookup_in_documentation` imported but unused

Remove unused import

(F401)